lib/RTCSession.js

```diff
@@ -676,7 +676,17 @@ module.exports = class RTCSession extends EventEmitter
         this._localMediaStream = stream;
         if (stream)
         {
-          this._connection.addStream(stream);
+          if (this._connecting.addTrack)
+          {
+            stream.getTracks().forEach(track => {
+              this._connection.addTrack(track, stream);
+            });
+          }
+
+          if (this._connection.addStream)
+          {
+            this._connection.addStream(stream);
+          }
```

Comment:
Why should it call `pc.addStream()` after having called `pc.addTrack()` before?

Reply:
For people having old implementation when listening event `addStream`, they do not have to change the code.

Reply:
`pc.onaddstream` event is not fired when calling `pc.addStream()` but just when calling `pc.setRemoteDescription()` with a remote SDP offer/answer that produces remote streams and tracks.

Reply:
Or should I check if `pc.addTrack()` exists, in case of that is not availble, fallback on `pc.addStream()` ?

Reply:
Probably, yes.